apps/arena/lib/arena/game/skill.ex

```diff
@@ -317,6 +317,37 @@ defmodule Arena.Game.Skill do
     |> put_in([:last_id], last_id)
   end
 
+  def do_mechanic(game_state, entity, {:polygon_hit, polygon_hit}, _skill_params) do
```

Comment:
This code is a bit similar to the to other damage mechanics, we should refactor this code later